app/src/main/java/com/sopt/now/compose/component/bottomnavigation/BottomNavigation.kt

```diff
@@ -0,0 +1,57 @@
+package com.sopt.now.compose.component.bottomnavigation
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Person
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.sopt.now.compose.R
+import com.sopt.now.compose.ui.theme.NOWSOPTAndroidTheme
+
+@Composable
+fun BottomNavigation(
+    selectedItem: Int,
+    onClick: (Int) -> Unit,
```

Comment:
```suggestion
    selectedItem: Int = 1,
    onClick: (Int) -> Unit = {},
```

요렇게 초기값을 줄 수도 있을 듯요 ????

Reply:
감사합니다!